app/Http/Controllers/WaitingRoomsController.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Gladiator\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Gladiator\Http\Controllers\Controller;
+use Gladiator\Models\WaitingRoom;
+
+class WaitingRoomsController extends Controller
+{
+    private $validationRules = [
+        'campaign_id' => 'required|numeric',
+        'campaign_run_id' => 'required|numeric',
+        'signup_start_date' => 'required|date',
+        'signup_end_date' => 'required|date'
+    ];
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $rooms = WaitingRoom::all();
+
+        return view('waitingrooms.index')->withRooms($rooms);
```

Comment:
Same with what @angaither mentioned in @deadlybutter's PR for competitions, could we switch to just using the `compact()` method?